app/controllers/documents_controller.rb

```diff
@@ -43,6 +43,22 @@ def destroy
     end
   end
 
+  def upload
+    @document = Document.new(creator_id: current_user.id, document: params[:documents])
+    if @document.save_and_verify
+      render_success_response
+    else
+      render_error_response
+    end
+  end
+
+  def delete
+    document = Document.find_by(id: params[:delete], creator: current_user, claim: nil)
```

Comment:
I think there might be an issue with the way we're setting `current_user` here. `current_user` is an instance of the `User` model, however the `creator` relation on the `Document` [model](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/jh-CTSKF-833-multi_upload/app/models/document.rb#L5) is to the `ExternalUser` model.

This could be changed to `creator: current_user.persona` here and elsewhere in this class, so that we're using the external user object relating to the user. I've done some brief testing and it doesn't seem to break the functionality, and I _think_ it will fix the failing tests we're currently seeing.